src/main/java/com/github/cypher/Main.java

```diff
@@ -1,23 +1,33 @@
 package com.github.cypher;
 
+import com.airhacks.afterburner.injection.Injector;
+import com.github.cypher.root.RootView;
 import javafx.application.Application;
-import javafx.fxml.FXMLLoader;
-import javafx.scene.Parent;
 import javafx.scene.Scene;
 import javafx.stage.Stage;
 
+import java.util.HashMap;
 import java.util.Locale;
-import java.util.ResourceBundle;
+import java.util.Map;
 
 public class Main extends Application {
     public static final String WORKING_DIRECTORY = ""; //The path to the folder where settings, credentials etc are saved.
 
     @Override
     public void start(Stage primaryStage) throws Exception {
-        Parent root = FXMLLoader.load(getClass().getResource("/fxml/root.fxml"),
-				ResourceBundle.getBundle("Cypher", new Locale("sv", "SE")));
+		Locale.setDefault(new Locale("sv", "SE"));
+        Map<Object, Object> customProperties = new HashMap<>();
+        customProperties.put("n1", 8);
```

Comment:
I guess those customProperties are the dependencies to be injected? You could maybe include a comment or two to make it more clear.